lib/tests/ebpd_loader_test.cc

```diff
@@ -0,0 +1,16 @@
+#include "gtest/gtest.h"
+#include "lib/tests/ebpf_sample.h"
+#include "lib/ebpd_loader.h"
+#include <iostream>
+#include <string_view>
+
+using namespace std;
+
+TEST(EbpdLoader, Test1) {
```

Comment:
You need a test For destruction of EbpdLoader() without calling LoadBuffer(). This would theoretically catch the problem with uninitialized handle that I mentioned above.

Reply:
This may not be required now, I guess.